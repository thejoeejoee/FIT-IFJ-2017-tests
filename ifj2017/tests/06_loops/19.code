' substr in loop

scope
    dim a as string
    dim a_len as integer
    dim res as string
    a = !"hgdfh_DFGkf5665678"
    a_len =  length(a)
    dim i as integer = 1

    do while i <= a_len
        res = substr(a, 1, i)
        print res;
        res = substr(a, i, a_len)
        print res;
        i = i + 1
    loop
    
end scope