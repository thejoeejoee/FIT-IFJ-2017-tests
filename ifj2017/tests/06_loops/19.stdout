hhgdfh_DFGkf5665678hggdfh_DFGkf5665678hgddfh_DFGkf5665678hgdffh_DFGkf5665678hgdfhh_DFGkf5665678hgdfh__DFGkf5665678hgdfh_DDFGkf5665678hgdfh_DFFGkf5665678hgdfh_DFGGkf5665678hgdfh_DFGkkf5665678hgdfh_DFGkff5665678hgdfh_DFGkf55665678hgdfh_DFGkf56665678hgdfh_DFGkf56665678hgdfh_DFGkf56655678hgdfh_DFGkf56656678hgdfh_DFGkf56656778hgdfh_DFGkf56656788